docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -238,6 +238,31 @@ will return a JSON object of the form:
 
 where 123456, 342332, etc are the candidates that exist for this project.
 
+#### 2.1.5 Single project electrophysiology recordings
+```
+GET /projects/$ProjectName/recordings
```

Comment:
"recordings" is ambiguous in this context.. how do people know it's EEG recordings and not, say, video recordings or some other type of recordings? 

I think this should be namespaced in some space (ie `eeg/recordings` or `electrophysiology/recordings`)

Reply:
Actually, I used recordings cause this could apply also to MEG and other electrophysiology recordings, not just EEG. I think at the beginning I had electrophysiology_recording but one of the comment early in the review process was that it was too long so I replaced it with recording...

Reply:
Is there some acronym you could use that encompasses MEG and EEG? Because I still think "recording" is too vague..

Reply:
I can't think of any at the moment. I will ask Christine if she thinks of another term that is generic but not too generic

Reply:
I'm not familiar with the field, but I did a brief search and most references just seems to say "EEG/MEG", so I guess I'll resolve this..